### DISCUSSION TITLE
poe 点击 log out 无法展示登出弹窗

### DISCUSSION THREAD

**Discussion by DeanWanghewei** [tw93/Pake#469] [Q&A]
poe 点击 log out 无法展示登出弹窗

**Comment by DeanWanghewei** [comment]
mac os 13.4

**Comment by tw93** [comment]
试了试可以的，辛苦用最新的包去试试
https://github.com/tw93/Pake/releases/latest/download/Poe.dmg

**Reply by DeanWanghewei** [reply]
OK,多谢.用最新的包可以点击退出了
